readme.md

```diff
@@ -1,11 +1,11 @@
-Street Facing - [Demo](http://alexhancock.github.com/street-facing/)
```

Comment:
Could you just change them to `https` links?, but leave the rest of the link the same.

It feels weird to have the content of the repo under my name linking to the github demo branch of a different user account.

I will update the gh-pages branch after you merge.